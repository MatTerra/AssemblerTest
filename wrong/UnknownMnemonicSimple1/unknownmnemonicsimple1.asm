            section data
            section text
start:                               ; Simple add something
            jmp     end
            jmpnz   end
; Let's finish this useless file
end:        stop