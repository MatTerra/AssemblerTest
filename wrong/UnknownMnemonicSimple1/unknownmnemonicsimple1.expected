Line 5 Unknown mnemonic jmpnz - Syntax
